bokeh/server/tornado.py

```diff
@@ -357,15 +416,6 @@ def log_stats(self):
             log.debug("[pid %d]   %s has %d sessions with %d unused",
                       os.getpid(), app_path, len(sessions), unused_count)
 
-    def keep_alive(self):
+    def _keep_alive(self):
         for c in self._clients:
             c.send_ping()
-
-    @gen.coroutine
-    def run_in_background(self, _func, *args, **kwargs):
-        """
-        Run a synchronous function in the background without disrupting
-        the main thread. Useful for long-running jobs.
-        """
-        res = yield self._executor.submit(_func, *args, **kwargs)
-        raise gen.Return(res)
```

Comment:
This was cruft that was never fully realized/implemented or used. Its not used by our code and it's not mentioned in any docs or issues, removing without comment.